.github/workflows/npm-cross-chain-arbitrum.yml

```diff
@@ -0,0 +1,58 @@
+name: NPM cross chain Arbitrum contracts
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "cross-chain/arbitrum/**"
+      - "cross-chain/arbitrum/deploy/**"
+      - "cross-chain/arbitrum/hardhat.config.ts"
+      - "cross-chain/arbitrum/package.json"
+      - "cross-chain/arbitrum/yarn.lock"
+  workflow_dispatch:
+
+jobs:
+  npm-compile-publish-contracts:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cross-chain/arbitrum
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14.x"
+          registry-url: "https://registry.npmjs.org"
+          cache: "yarn"
+          cache-dependency-path: cross-chain/arbitrum/yarn.lock
+
+      # We need this step because the `@keep-network/tbtc` which we update in
+      # next steps has a dependency to `@summa-tx/relay-sol@2.0.2` package, which
+      # downloads one of its sub-dependencies via unathenticated `git://`
+      # protocol. That protocol is no longer supported. Thanks to this step
+      # `https://` is used instead of `git://`.
+      - name: Configure git to not use unauthenticated protocol
+        run: git config --global url."https://".insteadOf git://
+
+      - name: Resolve latest contracts
+        run: |
+          yarn upgrade --exact @keep-network/tbtc
```

Comment:
I think we don't need this step. This package does not depend `@keep-network/tbtc`. If we remove it, we still need to install the dependencies though. That means we need to add:
```
- name: Install dependencies
  run: yarn install --frozen-lockfile
```

Reply:
I meant to add @keep-network/tbtc-v2. Changed.

Reply:
oops.. changed :)